debezium-platform-stage/src/utils/constants.ts

```diff
@@ -89,6 +89,16 @@ export const transformSchema = {
   required: ["name", "type", "schema", "config"],
 };
 
+export enum DatabaseType {
+  ORACLE = "ORACLE",
+  MYSQL = "MYSQL",
+  MARIADB = "MARIADB",
+  SQLSERVER = "SQLSERVER",
+  POSTGRESQL = "POSTGRESQL",
+}
+
+export const DEBEZIUM_SIGNAL_COLLECTION_NAME_SUFIX = "_debezium_signal";
```

Comment:
IMHO I would avoid to add this suffix and let the user choose the whole name. @Naros @jpechane WDYT?

Reply:
Yes, I agree, we have no such limitation at the core level. I am fine in filling it in input box as starting value but I don't think it should be limited this way.

Reply:
Update to remove the suffix from signal collection name.